exhibitors/views.py

```diff
@@ -58,6 +64,31 @@ def possible_contact_persons(fair):
     return contact_persons
 
 
+class UserAutocomplete(autocomplete.Select2QuerySetView):
+    def get_queryset(self):
+        # Don't forget to filter out results depending on the visitor !
+        if not self.request.user.is_authenticated:
+            return User.objects.none()
+
+        qs = (
+            User.objects.order_by("username")
+            .annotate(full_name=Concat("first_name", V(" "), "last_name"))
+            .all()
+        )
+
+        if self.q:
+            qs = qs.filter(
+                Q(email__istartswith=self.q)
```

Comment:
When a user is logged in though a KTH account, they have a `Profile` which sometimes contain their email and names. I think we need to use these fields in the filter as well. I could be wrong.

Reply:
The autocomplete is for the User model only, so we don't need to worry about Profile-specific attributes

Reply:
(And the many to many field in the exhibitor admin is of User objects only)

Reply:
Perfect!